### PR TITLE
Upgrade to Oracle JDBC driver v 21.3.0.0 and import its BOM

### PR DESCRIPTION
**Pull request by Sanne**

### CODE REVIEW

File: bom/application/pom.xml [hunk 1]
```diff
@@ -353,6 +353,15 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Oracle JDBC driver -->
+            <dependency>
+                <groupId>com.oracle.database.jdbc</groupId>
+                <artifactId>ojdbc-bom</artifactId>
+                <version>21.3.0.0</version>
```
Comment (gsmet) [inline bom/application/pom.xml:360]: Shouldn't you use the `${oracle-jdbc.version}` instead and update the property?

**Review by gsmet — CHANGES_REQUESTED** [review]

File: bom/application/pom.xml (hunk 1 above)
Comment (Sanne) [reply bom/application/pom.xml:360]: right, sure

Comment (Sanne) [reply bom/application/pom.xml]: done

**Review by gsmet — APPROVED** [review]

**Comment by Sanne** [comment]
The Data1 tests are failing because of https://github.com/quarkusio/quarkus/pull/19873#issuecomment-913659276

**Comment by manovotn** [comment]
I've sent https://github.com/quarkusio/quarkus/pull/19934 to address the CI problem.

**Pull request merged**